README.md

```diff
@@ -235,6 +241,25 @@ nok_data = dict(a=True, b=1)
 validate_data_spec(nok_data, StrictSpec) # raise Exception
 ```
 
+### Feature: Any Keys Set
+
+- A spec class decorated with e.g. `dsv_feature(any_keys_set={...})` means that at least one key among a keys tuple from the set must exist.
+```python
+from data_spec_validator.spec import Checker, validate_data_spec, dsv_feature, INT
+
+@dsv_feature(any_keys_set={('a', 'b')})
```

Comment:
Maybe show the example of multiple set combination and explain the AND validation logic instead of a single set in README?